back/spec/services/trending_idea_service_spec.rb

```diff
@@ -26,6 +26,18 @@
         expect(trending_filter.size).to eq(trending_filter_after.size + 1)
       end
     end
+
+    it 'does not include ideas that are native survey responses', document: false do
+      trending = nil
+      trending_filter = nil
+      travel_to Time.now do
+        trending = Idea.all.select { |i| described_class.new.trending? i }
+        generate_survey_response_ideas
+        trending_filter = described_class.new.filter_trending(Idea.all).map(&:id)
+      end
+
+      expect(trending_filter.size).to eq trending.size
```

Comment:
Maybe checking the actual items is more accurate than checking the size. And the same about the second test.

```suggestion
      expect(trending_filter).to match_array trending
```